### ISSUE TITLE
plot_timeseries problems

### ISSUE THREAD

**Issue by Karpefors** [Karpefors/Tendril#72]
Testing the timeseries plot I experience issues with its functionality:
1. The line through the events is broken up at several occations.
2. The actual net sum and days are not correct. Wrong columns used?

**Comment by stefanoborini** [comment]
The problem here is treatment of NaNs. plot_timeseries does a set of manipulations to obtain a dataset where each row represents a day. The dataset looks like this:

```
   Terms StartDay placebo active
1   AE33       10     NaN      1
2   AE33       14       1    NaN
3   AE33       31       1    NaN
4   AE33       42       1    NaN
5   AE33       53     NaN      1
6   AE33       64     NaN      1
7   AE33       96     NaN      1
8   AE33      101     NaN      1
9   AE33      134     NaN      1
```
This is of course because there's no value for the placebo at day 10, for example.


The problem arises from the fact that the plot displays the Net value that comes from the difference between the entries, which of course can't be computed for NaN values. 

Solution:
1. replace NaN with 0, or
2. remove rows with NaNs on either column.

I have the feeling that solution 1 is conceptually wrong, as it assumes a treatment result that has not physically taken place. Opinion?

**Comment by Karpefors** [comment]
I’m not sure that is the problem (solution 1 is already in use in the function). However, I suspected that it is the absolute reference to column numbers [e.g. time.grp <- dplyr::mutate_at(time.grp, 2:3, cum.na)] that causes the problem. If we have added a new columns, that could be a problem. And i think we might have done that when we made it possible to get e.g. demographic variables available after the tendril calculation.
